arduino/smartcar/smartcar.ino

```diff
@@ -513,14 +518,17 @@ void autoLeftPark(){ // the car is supposed to park inside a parking spot to its
         frontTimer++;
         distanceTraveled = newDistanceTraveled;
     }
-    Serial.println("done!");
-    car.setAngle(0);
-    car.setSpeed(PARKING_SPEED);
     // here the car will have the correct angle, car will drive foward and park
+    turningAngle = 0;
+    car.setAngle(turningAngle);
+    car.setSpeed(PARKING_SPEED);
+    while (!isObsAtFront(35)){
```

Comment:
This should be the same as the the while loop above at line 422.

Reply:
Fixed with https://github.com/DIT113-V22/group-02/pull/111/commits/590e4f481dd88a18fc2cef463aa02616c63bbbbb